docs/engineering/android-app/configuring/config-types/questionnaire.mdx

```diff
@@ -325,3 +326,43 @@ triggerConditions | This defines an array of condition for to be met for the eve
 eventResourceId |  uniqueId of resource id to be closed |  yes   |               |
 eventResources | A list of resources to close(Type of ResourceConfig) |    yes           |     |      |
 
+
```

Comment:
@ellykits  do we dictate where in the Patient, Group e.t.c resource is added?

Reply:
Which resources are you referring to here @dubdabasoduba? The resource used to sync unique identifiers? This new configuration (see the sample in the comment) is added to the Questionnaire config.